doc/faq.markdown

```diff
@@ -520,8 +520,7 @@ like this doesn't look good:
 		material
 	)
 
-That's why we added `makeSubscribe` at [fusion.flow](https://github.com/area9i
-nnovation/flow/blob/master/lib/fusion.flow). It has a more convenient syntax
+That's why we added `makeSubscribe` at [fusion.flow](https://github.com/area9innovation/flow/blob/master/lib/fusion.flow). It has a more convenient syntax
```

Comment:
Could you please change the link to "flow9" repo instead of "flow"?

Reply:
done